src/illustration/scamsEducation.tsx

```diff
@@ -0,0 +1,324 @@
+        fill="white"
+      />
+      <path
+        d="M98.4728 139.46L98.2446 140.338L97.6014 140.172L97.6567 139.958L94.6689 139.177L94.8418 138.486L98.4728 139.46Z"
+        fill="white"
+      />
+      <path
+        d="M99.005 137.74C98.7768 138.631 97.9676 138.949 96.7504 138.631C95.5331 138.313 94.9798 137.643 95.2081 136.752C95.4363 135.86 96.2524 135.55 97.4696 135.867C98.6869 136.185 99.2471 136.855 99.005 137.74ZM98.3826 137.581C98.4932 137.146 98.0921 136.766 97.2898 136.551C96.4876 136.337 95.9481 136.482 95.8374 136.918C95.7268 137.353 96.1279 137.74 96.9302 137.947C97.7325 138.155 98.265 138.016 98.3826 137.581Z"
+        fill="white"
+      />
+      <path
+        d="M95.2223 151.946L94.9941 152.824L94.3509 152.658L94.4062 152.444L91.4185 151.663L91.5914 150.972L95.2223 151.946Z"
+        fill="white"
+      />
+      <path
+        d="M95.7892 150.226C95.561 151.117 94.7518 151.435 93.5346 151.117C92.3173 150.799 91.764 150.129 91.9923 149.238C92.2205 148.346 93.0366 148.035 94.2538 148.353C95.4711 148.671 95.9898 149.341 95.7892 150.226ZM95.1668 150.067C95.2774 149.632 94.8763 149.251 94.074 149.037C93.2717 148.823 92.7323 148.968 92.6216 149.404C92.511 149.839 92.9121 150.226 93.7144 150.433C94.5166 150.64 95.0146 150.502 95.1322 150.067H95.1668Z"
+        fill="white"
+      />
+      <g opacity="0.48">
+        <path
+          d="M90.6026 133.843L99.3999 134.153L97.6155 157.053L87.9468 157.591L90.6026 133.843Z"
+          fill="#A33E61"
+        />
+      </g>
+      <path
+        d="M84.7236 156.901L97.7882 157.647L99.0054 134.796L85.9478 134.05L84.7236 156.901Z"
+        fill="#005156"
+      />
+      <path
+        d="M85.9541 156.002L87.2198 135.059L97.7738 135.695L96.515 156.638L85.9541 156.002ZM87.7246 135.625L86.5212 155.505L96.0102 156.072L97.2066 136.192L87.7246 135.625Z"
+        fill="white"
+      />
+      <path
+        d="M91.716 148.381C89.2123 148.229 87.3312 146.896 87.4211 145.341C87.5179 143.786 89.5443 142.687 92.048 142.839C94.5516 142.991 96.4328 144.325 96.336 145.88C96.2825 146.301 96.1179 146.7 95.859 147.036C95.6001 147.373 95.2563 147.634 94.8628 147.794C94.339 148.057 93.7762 148.234 93.196 148.319C92.706 148.388 92.2102 148.409 91.716 148.381ZM87.9605 145.375C87.8844 146.585 89.6204 147.718 91.7436 147.842C92.7302 147.927 93.7214 147.746 94.6138 147.317C94.9235 147.197 95.1963 146.998 95.405 146.74C95.6137 146.481 95.7511 146.173 95.8034 145.845C95.8726 144.636 94.1435 143.503 92.0134 143.372C91.5652 143.343 91.1152 143.364 90.6716 143.434C89.157 143.662 88.0158 144.415 87.9605 145.375Z"
+        fill="white"
+      />
+      <path
+        d="M94.2606 140.642V141.554H93.569V141.333L90.4844 141.243V140.552L94.2606 140.642Z"
+        fill="white"
```

Comment:
nit: Should we replace all "white" with "#FFF" for consistency?

Reply:
Or `{color.white}`